addons/mail/static/src/models/emoji_category.js

```diff
@@ -0,0 +1,38 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one, many } from '@mail/model/model_field';
+import { replace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'EmojiCategory',
+    identifyingFields: ['description'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeEmojiRegistry() {
+            return replace(this.messaging.emojiRegistry);
+        },
+    },
+    fields: {
+        description: attr({
+            readonly: true,
+            required: true,
+        }),
+        emoji: many('Emoji', {
```

Comment:
`emoji` => `emojis` (plural as it's a `many` field)

And even name it `allEmojis` for consistency and clarity